bagitobjecttransfer/recordtransfer/admin.py

```diff
@@ -224,23 +229,24 @@ class UploadSessionAdmin(ReadOnlyAdmin):
         - delete: Not allowed
     """
 
-    change_form_template = "admin/readonly_change_form.html"
+    @display(description=gettext("Upload Size"))
+    def upload_size(self, obj):
+        return get_human_readable_size(obj.upload_size, 1000, 2)
 
-    form = UploadSessionForm
+    fields = ["token", linkify("user"), "started_at", "file_count", "upload_size", "status"]
```

Comment:
Like I recommended for the uploaded file admin, some [search_fields](https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.search_fields) would be good here to be able to search upload sessions.

How about:

```python
search_fields = ["token", "user__username"]
```